js/app/src/components/Interpretation/InterpretationComponents/Image.svelte

```diff
@@ -67,8 +67,7 @@
 		<div class="interpretation">
 			<canvas bind:this={saliency_layer} />
 		</div>
-		<!-- svelte-ignore a11y-missing-attribute -->
-		<img bind:this={image} src={original} />
+		<img bind:this={image} src={original} alt={"uploaded input"} />
```

Comment:
Nit: do we need the curly brackets? I noticed you removed it from a different file
```suggestion
		<img bind:this={image} src={original} alt="uploaded input" />
```